aggregator/src/app/AggregationStrategy.ts

```diff
@@ -207,6 +213,7 @@ export default class AggregationStrategy {
   async #measureFees(bundles: Bundle[]): Promise<{
     success: boolean;
     fee: BigNumber;
+    errorReason: OperationResultError | null;
```

Comment:
We don't really use `null` internally - only when something external needs it. This should probably be documented somewhere. Can you make this `nil`?